tests/e2e_regression/expected/emerald_account_nfts.body

```diff
@@ -221,12 +755,9 @@
       }
     },
     {
-      "description": "",
```

Comment:
these fields are removed by following the logic posted in the PR:
- we can't get metadata from this URL (unresolvable hostname)
- so the name+description+image are left as nil in the EVMNFTData struct during analysis
- which go in as NULL in the database
- which I suppose we are leaving them as field-absent in our API output :shrug: